components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -372,8 +372,7 @@ function SpendingLimitReachedModal(p: { hints: any }) {
     const [attributedTeam, setAttributedTeam] = useState<Team | undefined>();
 
     useEffect(() => {
-        const attributionId: AttributionId | undefined =
-            p.hints && p.hints.attributionId && AttributionId.parse(p.hints.attributionId);
+        const attributionId: AttributionId | undefined = p.hints && p.hints.attributionId && p.hints.attributionId;
```

Comment:
Below, there is a check on `attributionId.kind` -- meaning this should be an object, resulting from a `parse`, right?

I don't think using the `string` `p.hints.attributionId` makes sense (also you're using it twice here).